packages/design-picker/src/components/generated-design-picker.tsx

```diff
@@ -3,6 +3,7 @@
 import { Button } from '@automattic/components';
 import { MShotsImage } from '@automattic/onboarding';
 import { useViewportMatch } from '@wordpress/compose';
+import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
```

Comment:
I have not found a way to import `sprintf` from `@wordpress/react-i18n`.

Reply:
Yes, it's in `@wordpress/i18n`